examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>This is just my personal opinion but I think doing all the import at once at the beginning and explaining why we will need them makes tutorials easier to follow. Jupyter notebooks can get messy real quicly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I'm actually a fan of doing the imports only when they're necessary, so that the context for them is sort of immediately obvious. But I don't feel super strongly about it, so I'm happy either way!</p>